spyder/preferences/shortcuts.py

```diff
@@ -81,9 +106,18 @@ def keyReleaseEvent(self, e):
         """Qt Override"""
         self.parent().keyReleaseEvent(e)
 
+    def setText(self, sequence):
+        """Qt method extension."""
+        if sys.platform == 'darwin':
+            sequence = sequence.replace('Meta', 'Ctrl')

```

Comment:
The official name of the key on all Mac keyboards as well as most if not all the Mac literature is ``Control``, not ``Ctrl``. Furthermore, this also makes it more likely to be confused with ``Cmd``, since the general convention is to translate ``Ctrl`` (the PC/Linux key) --> ``Cmd``. Therefore, that should be used, unless some technical restriction prevents it.

Reply:
I do not know Mac at all, so I'll take your words for it. I wish we had someone with a Mac on the team to test stuff....

Reply:
I grew up a big Mac user, but I switched to Windows (with some use of Linux) in high school since I couldn't afford to buy my own Mac laptop, and particularly with Apple's increasing "smartphonification" and the inability to build my own machine, there's little chance of me ever switching back.

I suppose we could consider using some of our $$$ and purchasing e.g. a cheap MacBook Pro (>= mid-2012, so its compatible with 10.14+) for one of the devs to test on, but that would be [at least $300-400](https://www.ebay.com/sch/i.html?_sop=12&Release%2520Year=2012%7C2013%7C2014&LH_ItemCondition=3000%7C2500%7C2000%7C1500&Processor%2520Type=Intel%2520Core%2520i5%25203rd%2520Gen%252E%7CIntel%2520Core%2520i5%25204th%2520Gen%252E%7CIntel%2520Core%2520i7%25203rd%2520Gen%252E&_fsrp=1&_nkw=MacBook+Pro&_sacat=111422&LH_Complete=1&rt=nc&_dcat=111422&LH_Sold=1).